test/ci-operator-integration/base/run.sh

```diff
@@ -35,6 +39,7 @@ run_test() {
         --determinize-output \
         --namespace "${TEST_NAMESPACE}" \
         --config "${TEST_CONFIG}" \
+        --image-import-pull-secret "${PULL_SECRET_PATH}" \
```

Comment:
What you've done here is the same as what you did in the Go tests. Each invocation of `run_test` in this script is analogous to a test case, and when you add this arg to every single possible test invocation, you're muddying the waters. A good test will test something in a targeted manner. Imagine what happens when someone makes a change that breaks this functionality -- they will run the tests and see every single integration test fail. Then, they'll read what the tests are doing and try to figure out why their change had that effect. Instead, let's add a new test case here that specifically sets this flag so if someone breaks it in the future, they can see one isolated test case (that's clearly labeled as testing this feature) fail.

Reply:
Ack. Will add a new case.

Reply:
I guess a new case will be a new folder under `./test/ci-operator-integration`

Reply:
Done.